repose-aggregator/functional-tests/spock-functional-test/src/test/configs/features/filters/valkyrie/system-model.cfg.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<system-model xmlns="http://docs.openrepose.org/repose/system-model/v2.0">
+  <repose-cluster id="repose">
+
+    <nodes>
+      <node id="simple-node" hostname="localhost" http-port="${reposePort}"/>
+    </nodes>
+
+    <filters>
+      <filter name="client-auth"/>
+      <!--<filter name="api-validator"/>-->
```

Comment:
If we're not going to use the API-validator stuff, we should take out the files...